contracts/src/IRollup.sol

```diff
@@ -200,17 +201,8 @@ interface IRollup {
      *
      * @param vmHash New VM hash.
      * @param inboxSize Size of inbox corresponding to assertion (number of transactions).
-     * @param l2GasUsed Total L2 gas used as of the end of this assertion's last transaction.
-     * @param prevVMHash Predecessor assertion VM hash (required because it does not get stored in the assertion).
-     * @param prevL2GasUsed Predecessor assertion L2 gas used (required because it does not get stored in the assertion).
      */
-    function createAssertion(
-        bytes32 vmHash,
-        uint256 inboxSize,
-        uint256 l2GasUsed,
-        bytes32 prevVMHash,
-        uint256 prevL2GasUsed
-    ) external;
+    function createAssertion(bytes32 vmHash, uint256 inboxSize) external;
```

Comment:
can be fetched directly from `parent.stateHash` since stateHash is just the vmHash now (only need to provide it if the structure of `stateHash` is modified)